Editing cave diving survey (#1)
zheylmun/site: content/posts/cave_diving_survey.md

#### MNemo Survey

> MNemo survey also involves taking distance and depth measurementa and a compass heading at each station,

Reviewer: ```suggestion
MNemo survey also involves taking distance and depth measurements and a compass heading at each station,
```

Both manual and MNemo survey share a similar set of challenges.

* Measurement Error: All measurements have some degree of error which can be caused by a variety of factors,
  including human error, equipment error, and environmental conditions.

>   In particular, the line used to represent a straight line between stations may stretch, sag, or be affected by currents, resulting in inaccurate distance and/or heading measurements.

Reviewer: ```suggestion
  In particular, the cave line or measuring tape between stations may stretch, sag, or be affected by currents, resulting in inaccurate distance and/or heading measurements.
```
